plugins/nodes/src/js/columns/__tests__/__snapshots__/NodesTableHealthColumn-test.js.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`NodesTableHealthColumn renderer renders with health status 'Healthy' 1`] = `
 <div
-  className="css-19m5nce"
+  className="css-4tf7uh"
```

Comment:
Did you change anything at the nodes column? Or is this cause by the UI-KIt update?

Reply:
This is a snapshot update, whenever there is a build the `css-randomString` changes and the snapshots need to be updated

Reply:
yes could be because of the ui-kit update.